src/constants.ts

```diff
@@ -1,3 +1,8 @@
 export const STORE_DEFAULT_NAME = '__LSM__';
 export const STORE_ACTION_NAME = '__LSM_NAME__';
-export const PERSIST_BEFORE_UNLOAD = 'beforeUnload';
+
+export enum PersistOption {
+  None = 'none',
+  OnAction = 'onAction',
+  BeforeUnload = 'beforeUnload',
+}
```

Comment:
```suggestion
export const PersistOption = {
  none: 'none',
  action: 'action',
  unload = 'unload',
} as const
```

save some bytes over enum

Reply:
So we don't need backward compatibility here and we can use these shorter option names? "none", "action", "unload"?